### PR TITLE
Issue #1: Update README.md to include project details

### LINKED ISSUE #1: Update README.md to include project info and setup steps.
Update [README.md](https://github.com/LNMIIT-CODING/STONKS-FrontEnd/blob/main/README.md) to include Project information, how to setup project, how to contribute, project tech stack, list of moderators to approach help for specific issue. Modify current [README.md](https://github.com/LNMIIT-CODING/STONKS-FrontEnd/blob/main/README.md) and remove redundant data.

#### Basic Techincal Info
1. Project Based on **React JS**
2. We would be using **Yarn** to setup the project.
3. Tech Stack would include **React JS, PostgreSQL** for now.

EDIT :- List of moderators to approach would be shared shortly.

### PR DESCRIPTION
**Pull request by Ayush-Shukla02**
-> Added the required headings
-> Added list of mods
-> Added the members list
-> Added Project Tech Stack

### CODE REVIEW

**Comment by AkMo3** [comment]
Please add a brief discription of the project, like what is our aim, what this project aims to do. Some key points to include
1. This project is frontend part of the STONKS application.
2. Stonks is app that will have capability to predict stock movement based of LR model.
3. Users are able to see current stock parameters. 
4. In future crypto may also be supported. 
 
@DreadArceus @OmGuptaIND Please suggest points if I forgot to mention mention something.
Take full benefit of TCE classes, lets see how many classes you have attended.
Also, reduce the size of [setting-up-project](https://github.com/Ayush-Shukla02/stonks-frontend#setting-up-project) and contribution.

File: README.md [hunk 1]
```diff
@@ -1,70 +1,88 @@
-# Getting Started with Create React App
+# Project Information
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-
-## Available Scripts
 
-In the project directory, you can run:
+    This project is frontend part of the STONKS application.
```
Comment (AkMo3) [inline README.md]: Same to same nahi tape na tha. Paragraph mai bana deta. Documentation mai to english use kar le :)

File: README.md [hunk 2]
```diff
@@ -1,70 +1,88 @@
...
+* [Souhardya Malakar](https://github.com/SouhardyaMalakar)
+* [Sourabh](https://github.com/sourabhjo7)
+* [Sriraj Behera](https://github.com/SrirajBehera)
+* [Utkarsh Choubey](https://github.com/utk2541)
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+# Setting up Project
```
Comment (AkMo3) [inline README.md]: Reduce size to header3 i.e instead of single hash make it `###`

File: README.md [hunk 3]
```diff
@@ -1,70 +1,88 @@
...
 
-### `yarn eject`
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+# Contribution
```
Comment (AkMo3) [inline README.md]: Same as above, reduce size.

File: README.md [hunk 4]
```diff
@@ -1,70 +1,88 @@
...
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+## Getting Started with Create React App
```
Comment (AkMo3) [inline README.md]: Adds the steps of contribution here shared with you [here](https://github.com/LNMIIT-CODING/stonks-frontend/issues/1#issuecomment-804931213)

File: README.md [hunk 5]
```diff
@@ -1,70 +1,88 @@
...
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Learn More
+### Available Scripts
```
Comment (AkMo3) [inline README.md]: Change available scripts to `Some useful scripts`

File: README.md [hunk 6]
```diff
@@ -1,70 +1,88 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+The page will reload if you make edits.\
+You will also see any lint errors in the console.
 
-### Analyzing the Bundle Size
+#### `yarn test`
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
```
Comment (AkMo3) [inline README.md]: Remove this line.

File: README.md [hunk 7]
```diff
@@ -1,70 +1,88 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
 
-### Advanced Configuration
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
```
Comment (AkMo3) [inline README.md]: Remove this line.

File: README.md [hunk 8]
```diff
@@ -1,70 +1,88 @@
...
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
-
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+## List of Moderators
 
-### `yarn test`
+* **[Mayank Talwar](https://github.com/DreadArceus)**
```
Comment (AkMo3) [inline README.md]: Add everything to him that others contain except GitHub. That is exclusively for me :)

File: README.md [hunk 9]
```diff
@@ -1,70 +1,88 @@
...
+* **[Akash Mondal](https://github.com/AkMo3)**: GitHub
+* **[Om Gupta](https://github.com/OmGuptaIND)**: React, Node.js, Django REST, MongoDB, PostgreSQL, SQLite
+* **[Souhardya Malakar](https://github.com/SouhardyaMalakar)**: CSS, React
+* **[Arush Kurundodi](https://github.com/arcinston)**
```
Comment (AkMo3) [inline README.md]: Add react to him.

File: README.md [hunk 10]
```diff
@@ -1,70 +1,88 @@
...
+* **[Sriraj Behera](https://github.com/SrirajBehera)**: JavaScript, Node, MongoDB
+* **[Utkarsh Choubey](https://github.com/utk2541)**
+* **[Ananjan](https://github.com/ananjan)**
+* **[Shivral](https://github.com/shivral)**: Web Scraping
+* **[Siddhant Singhania](https://github.com/sid11428)**
+* **[Sourabh](https://github.com/sourabhjo7)**
```
Comment (AkMo3) [inline README.md]: Leave them as it is right now, I am not aware of their hidden skill set

**Review by AkMo3 — CHANGES_REQUESTED** [review]
Points

**Comment by AkMo3** [comment]
Also, all the conversation would happen here, no more conversation in issue.

File: README.md (hunk 1 above)
Comment (Ayush-Shukla02) [reply README.md]: Are han maine bas copy kara tha ki yaha edit kardunga. usmein abhi points add kardunga aur bhi
same waise bhi na rakhta main

File: README.md (hunk 9 above)
Comment (Ayush-Shukla02) [reply README.md]: Done

File: README.md (hunk 8 above)
Comment (Ayush-Shukla02) [reply README.md]: Done

File: README.md (hunk 7 above)
Comment (Ayush-Shukla02) [reply README.md:81]: Both these lines, red and green both?

File: README.md (hunk 5 above)
Comment (Ayush-Shukla02) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (Ayush-Shukla02) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (Ayush-Shukla02) [reply README.md]: Done

File: README.md (hunk 7 above)
Comment (AkMo3) [reply README.md]: Line 81

File: README.md (hunk 5 above)
Comment (AkMo3) [reply README.md]: Not done. Push changes if done locally

File: README.md (hunk 3 above)
Comment (AkMo3) [reply README.md]: Not done. Push changes if done locally

File: README.md (hunk 2 above)
Comment (AkMo3) [reply README.md]: Not done. Push changes if done locally

File: README.md (hunk 9 above)
Comment (AkMo3) [reply README.md]: Not done. Push changes if done locally

File: README.md (hunk 8 above)
Comment (AkMo3) [reply README.md]: Not done. Push changes if done locally

File: README.md (hunk 6 above)
Comment (Ayush-Shukla02) [reply README.md]: Here only line 71 right as some info is to be given for yarn test?

File: README.md (hunk 7 above)
Comment (Ayush-Shukla02) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (Ayush-Shukla02) [reply README.md]: Done

File: README.md (hunk 6 above)
Comment (AkMo3) [reply README.md]: yup only 71

File: README.md [hunk 11]
```diff
@@ -1,70 +1,89 @@
...
+* Take up an issue
+* Write the code
+* Commit the changes made in a new branch
+* Make a PR(Pull Request)
 
-## Learn More
+## Getting Started with Create React App
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (AkMo3) [inline README.md]: Ye dono line hata de.

File: README.md [hunk 12]
```diff
@@ -1,70 +1,89 @@
...
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+### Setting up Project
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+### Contribution
```
Comment (AkMo3) [inline README.md]: Isko `steps to contribute` kar de.

File: README.md [hunk 13]
```diff
@@ -1,70 +1,89 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
 
-### Deployment
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+# Project Tech Stack
```
Comment (AkMo3) [inline README.md]: Size chota karde iska. H4 karde.

**Review by AkMo3 — CHANGES_REQUESTED** [review]
Minor:

File: README.md (hunk 6 above)
Comment (Ayush-Shukla02) [reply README.md]: Done

File: README.md (hunk 12 above)
Comment (Ayush-Shukla02) [reply README.md]: Done

File: README.md (hunk 13 above)
Comment (Ayush-Shukla02) [reply README.md]: Done

File: README.md (hunk 11 above)
Comment (Ayush-Shukla02) [reply README.md]: Done

File: README.md [hunk 14]
```diff
@@ -1,70 +1,86 @@
...
+* [Siddhant Singhania](https://github.com/sid11428)
+* [Souhardya Malakar](https://github.com/SouhardyaMalakar)
+* [Sourabh](https://github.com/sourabhjo7)
+* [Sriraj Behera](https://github.com/SrirajBehera)
+* [Utkarsh Choubey](https://github.com/utk2541)
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
 
-### `yarn eject`
+### Setting up Project
```
Comment (AkMo3) [inline README.md]: Isko H2 kar de.

**Review by AkMo3 — CHANGES_REQUESTED** [review]
Last one:

File: README.md (hunk 14 above)
Comment (Ayush-Shukla02) [reply README.md]: Done.
Also list of mods and list of members ko H3 kardu kya? thoda zaada bada nahi lag rha?

Comment (AkMo3) [reply README.md]: Ha.

Comment (Ayush-Shukla02) [reply README.md]: Done. 
I will add the Project Information in some time.

**Comment by AkMo3** [comment]
Ab main part. Tujhe sare commits to squash karke ek mai karna hai. Google karle kaise karta hai, agar nahi mila to ping kar dio.

File: README.md [hunk 15]
```diff
@@ -47,6 +47,7 @@ In future Crypto may also be included and emphasis will be given on improving th
 
 ### Steps to Contribute
 * Fork the Repo
+* Clone the forked repo
```
Comment (OmGuptaIND) [inline README.md:50]: Also, add to make a new branch the steps should be as follows:

1. Fork the repo
2. Clone the forked repo
3. Checkout from the master branch to a new branch.
4. Take an issue
5. Commit changes to the new branch for every issue 
6. Commit the changes to the new branch
7. Make a PR request from the new branch

**Review by OmGuptaIND — CHANGES_REQUESTED** [review]

File: README.md (hunk 15 above)
Comment (AkMo3) [reply README.md:50]: May be something like this is even better
1. Fork the repo
2. Clone the forked repo
3. Take an issue
4. Checkout to a new branch for every issue
5. Commit the changes to the new branch and squash them to be one in right format.
6. Commit message format is **Issue `#ISSUE_NO`: `SINGLE LINE DISCRIPTION OF ISSUE`**
   Eg: Issue #1: Update README.md to include project details
7. Make a PR request from the new branch

Comment (Ayush-Shukla02) [reply README.md:50]: Done

Comment (AkMo3) [reply README.md:50]: Ayush please see the new list by me.

Comment (AkMo3) [reply README.md:50]: @Ayush-Shukla02 ping

Comment (Ayush-Shukla02) [reply README.md:50]: is the example also needed in point 6?

Comment (AkMo3) [reply README.md:50]: Yup, it would make things clear for everyone. And you can use this exact commit message for this PR.

Comment (Ayush-Shukla02) [reply README.md:50]: Done

Comment (AkMo3) [reply README.md:50]: Great nice work. Just waiting for squashing of commits. Reset is done.

Comment (Ayush-Shukla02) [reply README.md:50]: Yes I have cloned it on my laptop. I will see how to do it.

Comment (Ayush-Shukla02) [reply README.md:50]: Just working under the guidance of you all. 😊

**Review by AkMo3 — APPROVED** [review]

**Review by DreadArceus — CHANGES_REQUESTED** [review]
Add Tensorflow(Keras) and Numpy to Moderator tech stacks for Ananjan, me and Souhardya.
Also add React to Siddhant and Utkarsh

**Review by DreadArceus — APPROVED** [review]

**Pull request merged**